### PR TITLE
Fix helm template

### PR DESCRIPTION
**Pull request by tiagolobocastro**
fix(helm): incompatible types for comparison

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

ci: validate the default template

Adds test for jenkins and gha to ensure we don't submit an invalid template.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

**Comment by tiagolobocastro** [comment]
This should fail:
bors try

**Comment by tiagolobocastro** [comment]
Failing for the wrong reasons :)
bors try

**Comment by tiagolobocastro** [comment]
bors try

**Review by r1jt — APPROVED** [review]
LGTM

File: scripts/helm/test-template.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+die()
```
Comment (avishnu) [inline scripts/helm/test-template.sh:3]: How does this get invoked?

Comment (tiagolobocastro) [reply scripts/helm/test-template.sh:3]: it doesn't, copy&pasto, I'll remove it

**Comment by tiagolobocastro** [comment]
bors merge

**Review by avishnu — APPROVED** [review]
Lgtm

**Pull request merged**